src/execution/sql/storage_interface.cpp

```diff
@@ -82,4 +84,13 @@ void StorageInterface::IndexDelete(storage::TupleSlot table_tuple_slot) {
   curr_index_->Delete(exec_ctx_->GetTxn(), *index_pr_, table_tuple_slot);
 }
 
+bool StorageInterface::IndexCreateInsert(catalog::index_oid_t index_oid, storage::TupleSlot table_tuple_slot) {
+  terrier::storage::index::IndexBuilder index_builder;
```

Comment:
`IndexBuilder` is not for inserting into indexes. I'll expand this in a future comment.

Reply:
Now IndexBuilder works as it originally did